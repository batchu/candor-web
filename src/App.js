import React,{Component} from 'react'
import Login from './Login'
import Dashboard from './Dashboard'
import Settings from './Settings'
import {BrowserRouter as Router,
Switch,
Route,
Link} from 'react-router-dom'

class App extends React.Component{

    render(){
        return <Router>
            <Switch>
                <Route path="login">
                    <Login />
                </Route>
                <Route path="/dashboard">
                    <Dashboard />
                </Route>
                <Route path="">
                    <Login />
                </Route>
            </Switch>
        </Router>
    }
}




export default App
