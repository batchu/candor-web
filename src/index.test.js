import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';


it('renders correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  
  /*

  function add(x,y) => x+y+1

  Test case 1:
  x=2
  y=3
  Expected output = 5
  Actual output =6
  Test case will fail
  */