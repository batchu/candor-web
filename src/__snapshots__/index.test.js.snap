// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <nav
    className="navbar navbar-expand-lg navbar-light fixed-top"
  >
    <div
      className="container"
    >
      <a
        className="navbar-brand"
        href="/login"
        onClick={[Function]}
      >
        Candor
      </a>
      <div
        className="collapse navbar-collapse"
        id="navbarTogglerDemo02"
      >
        <ul
          className="navbar-nav ml-auto"
        >
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/login"
              onClick={[Function]}
            >
              Login
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
  <div
    className="auth-wrapper"
  >
    <div
      className="auth-inner"
    >
      <form>
        <h3>
          Login
        </h3>
        <button
          class="loginBtn loginBtn--facebook"
        >
          Login with Facebook
        </button>
        <button
          class="loginBtn loginBtn--google"
        >
          Login with Google
        </button>
      </form>
    </div>
  </div>
</div>
`;
